huaweicloud-sdk-python-modelarts-dataset/modelarts/test/test_manifest_write_classification.py

```diff
@@ -37,26 +37,27 @@ def create_manifest():
         annotation_name = "Dog"
       annotation_creation_time = "2019-02-20 08:23:06"
       annotation_format = "manifest"
+      annotation_property = {"color": "black"}
       annotation_confidence = 0.8
       annotated_by = "human"
       annotations_list.append(
         Annotation(name=annotation_name, type=annotation_type,
                    confidence=annotation_confidence,
                    creation_time=annotation_creation_time,
-                   annotated_by=annotated_by, annotation_format=annotation_format))
+                   annotated_by=annotated_by, annotation_format=annotation_format, property=annotation_property))
```

Comment:
Consider to rename the parameter name `annotated_by`. Use `annotated_source` ?

Reply:
It's ok for changing this in this project, but other service need change too. Can we discuss it in a small meeting?